motor/discovery/gcp/mql_asset_objects.go

```diff
@@ -38,6 +45,82 @@ func getTitleFamily(o gcpObject) (gcpObjectPlatformInfo, error) {
 	return gcpObjectPlatformInfo{}, errors.Newf("missing runtime info for gcp object service %s type %s", o.service, o.objectType)
 }
 
+func computeInstances(m *MqlDiscovery, project string, tc *providers.Config, sfn common.QuerySecretFn) []*asset.Asset {
+	assets := []*asset.Asset{}
+	instances := m.GetList("return gcp.project.compute.instances { id name labels zone status networkInterfaces disks { guestOsFeatures } }")
```

Comment:
Apparently this thing `m.GetList` silently ignores errors. If you mess up the query it will just return an empty list and you won't even notice it actually does not work.
cc @vjeffrey